system/settings/settings.xml

```diff
@@ -463,6 +463,16 @@
           <default>false</default>
           <control type="toggle" />
         </setting>
+        <setting id="videoplayer.commskipenabled" type="boolean" label="38017" help="38018">
+          <level>1</level>
```

Comment:
this in no way a lowest level setting. People who make EDL list are advanced users